src/registrationTracking.js

```diff
@@ -0,0 +1,94 @@
+import{pushInDataLayer} from './modules/pushInDataLayer';
+import{checkEmptyField} from './modules/checkEmptyField';
+
+window.addEventListener('DOMContentLoaded', () => {
+    const regForm = document.getElementById('regForm');
+    const errorsArray = Array.from(document.querySelectorAll('.field-validation-error'));
+    let errorMessages = [];
+    let errorString;
+
+    const nameField = document.getElementById('Name'),
+        nameLabel = nameField.getAttribute('name');
```

Comment:
I'd consider introducing a `try/catch` around these `.getAttribute()` calls - just in case anyone changes the `id` attribute of any of the inputs. It will also help to log something helpful for later debugging.

```javascript
try {   
 const nameField = document.getElementById('Name'),
        nameLabel = nameField.getAttribute('name');
\\ ... all the others
} catch(e) {
 console.error(`Error in registrationTracking.js: ${e}`);
}
```

Reply:
Done 👍